src/commands.c

```diff
@@ -3804,6 +3804,20 @@ struct redisCommandArg BGSAVE_Args[] = {
 /* COMMAND COUNT hints */
 #define COMMAND_COUNT_Hints NULL
 
+/********** COMMAND DETAILS ********************/
```

Comment:
if we decide in advance that this subcommand contains only doc info - maybe we should call it COMMAND DOCS?

Reply:
good point. i'll add it to the top comment as a todo, and ask the core-team to comment.